src/schema-generator/types.ts

```diff
@@ -81,6 +82,7 @@ interface IRule {
 	excludes?: unknown | undefined;
 	uinfin?: boolean | undefined;
 	equalsField?: unknown | undefined;
+	withinDays?: IWithinDaysRule | undefined;
```

Comment:
move this to `IRenderRule` because `IRule` is used by all fields for validation when in fact only `date-field` is meant to have the `withinDays` validation rule